packages/synapse-react-client/src/components/EntityFinder/EntityFinder.tsx

```diff
@@ -359,20 +359,22 @@ export function EntityFinder({
             <div style={searchActive ? { display: 'none' } : {}}>
               {treeOnly ? (
                 <div>
-                  <EntityTree
-                    toggleSelection={toggleSelection}
-                    showDropdown={true}
-                    visibleTypes={selectableAndVisibleTypesInTree}
-                    initialScope={initialScope}
-                    selectedEntities={selectedEntities}
-                    projectId={projectId}
-                    initialContainer={initialContainer}
-                    currentContainer={currentContainer}
-                    setCurrentContainer={setCurrentContainer}
-                    showScopeAsRootNode={false}
-                    treeNodeType={EntityTreeNodeType.SINGLE_PANE}
-                    selectableTypes={selectableTypes}
-                  />
+                  <SynapseErrorBoundary>
```

Comment:
Added a bunch of error boundaries just to help isolate the problem to figure out where it was coming from